css-typed-om/README.md

```diff
@@ -0,0 +1,116 @@
+# CSS Typed OM Explained
+
+CSS Typed OM is the extensible API for the CSSOM that reduces the burden of manipulating a CSS property's value via string manipulation. It does so by exposing CSS values as typed JavaScript objects rather than strings.
+
+## Motivation
+
+The benefits of CSS Typed OM include:
+* Allows the performant manipulation of values assigned to CSS properties. 
+* Being able to write more maintainable code:
+	* Your code is easier to understand,
+	* Easier to write more generic code.
+
+### Examples
+
+### Getting a property's value
+
+In CSSOM land:
+To get specified and computed values for CSS properties of an element in CSSOM we get them via accessing them of the `.style` attribute on an element and the `getComputedStyle()` object respectively. 
```

Comment:
maybe a comma between "in CSSOM" and "we get them"?

Reply:
Changed